### ISSUE TITLE
Define Dataset: Address Points (Geospatial)

### ISSUE THREAD

**Issue by emily878** [sunlightpolicy/State-Open-Data-Census#35]
Define the essential substantive elements of the core Address Points dataset. What are the components that it must minimally include? Do we have a dataset that we could hold up as a model?

**Comment by waldoj** [comment]
@iandees, is [OpenAddress' list of attribute tags](https://github.com/openaddresses/openaddresses/blob/master/CONTRIBUTING.md#attribute-tags) a good reference for the minimal components of an address points dataset?

**Comment by iandees** [comment]
Yep, bare minimum is house number and street name. Adding city and postcode makes it more useful. Even better would be to have street name split into prefix, road name, suffix, and road type.

**Comment by waldoj** [comment]
I hadn't realized that the minimum could be so minimal. Thanks, Ian!

**Comment by iandees** [comment]
The other fields can be backfilled/estimated from other datasets, so it's a whole lot more work, but still useful.

**Comment by waldoj** [comment]
So, to be explicit, here's our minimum viable dataset:
- address: numeric component
- address: street name
- coordinates
